src/widgets/current-refined-values/current-refined-values.js

```diff
@@ -233,7 +233,7 @@ function clearRefinementFromState(state, refinement) {
   case 'exclude':
     return state.removeExcludeRefinement(refinement.attributeName, refinement.name);
   case 'numeric':
-    return state.removeNumericRefinement(refinement.attributeName, refinement.operator, refinement.name);
```

Comment:
> Well it's badly named for sure. I don't remember why though. Would it be possible to change the name of the property to value and leave the value as a number?

The thing is that in every template we pass to * refinement list (menus, hierarchical ..), we always use "name" that's why it was set as name, for consistency and ease of use.